sartography/spiff-arena
# Feature/avoid repredicting tasks

This branch removes code that would predict the state of child tasks again from within the backend. This caused issues where cycle timer boundary events were being set to MAYBE when spiff had already correctly set them to WAITING. We do not think this code is actually doing anything useful anymore so just ignore it.

To test:
1. Ensure process models with timer boundary events (cycle and duration) can run without error
